From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Adithya R <gh0strider.2k18.reborn@gmail.com>
Date: Sun, 16 Jan 2022 15:50:04 +0530
Subject: [PATCH] arm64: configs: moto-holi: Enable
 CC_OPTIMIZE_FOR_PERFORMANCE_O3

Allow all architectures to enable it.

Change-Id: Ia8ba5b77fd1b6ff2603cac13ea34221acca8e8dc
Signed-off-by: Sevenrock <sevenrock@hotmail.de>
---
 arch/arm64/configs/vendor/lineage_moto-holi.config | 1 +
 init/Kconfig                                       | 1 -
 2 files changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/configs/vendor/lineage_moto-holi.config b/arch/arm64/configs/vendor/lineage_moto-holi.config
index 5cc4db6202e1..cac6fc2c17f8 100644
--- a/arch/arm64/configs/vendor/lineage_moto-holi.config
+++ b/arch/arm64/configs/vendor/lineage_moto-holi.config
@@ -81,3 +81,4 @@ CONFIG_NR_CPUS=8
 # CONFIG_UNMAP_KERNEL_AT_EL0 is not set
 # CONFIG_HARDEN_BRANCH_PREDICTOR is not set
 # CONFIG_KALLSYMS_ALL is not set
+CONFIG_CC_OPTIMIZE_FOR_PERFORMANCE_O3=y
diff --git a/init/Kconfig b/init/Kconfig
index 3d16587c5e90..6b5521f6e7ab 100644
--- a/init/Kconfig
+++ b/init/Kconfig
@@ -1276,7 +1276,6 @@ config CC_OPTIMIZE_FOR_PERFORMANCE
 
 config CC_OPTIMIZE_FOR_PERFORMANCE_O3
 	bool "Optimize more for performance (-O3)"
-	depends on ARC
 	help
 	  Choosing this option will pass "-O3" to your compiler to optimize
 	  the kernel yet more for performance.
-- 
2.39.5

